deckhand/control/api.py

```diff
@@ -15,16 +15,46 @@
 import os
 
 import falcon
+from oslo_config import cfg
+from oslo_log import log as logging
 
+from deckhand.conf import config
 from deckhand.control import base as api_base
 from deckhand.control import secrets
 
+CONF = cfg.CONF
+
+
+def __setup_logging():
```

Comment:
This might be an Openstack pattern I'm not familiar with, but why do we need a separate config file for logging?